recipes/gcm-filters/LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+Copyright 2021 GCM Filters Developers
```

Comment:
Since this is already included in the PyPI tarball, better to rely on getting it from there rather than including it in the recipe. Can you remove from the PR?

Reply:
Ok thanks for that. I guess I was misinterpreting the "License file is packaged" checkbox to mean we should include it here.

Reply:
Yeah, that language is slightly ambiguous. It means "make sure it appears in the end result". Essentially, point `license_file` to the name of the file. If it's in the tarball, great! Otherwise, it will error and you need add it manually to the recipe (and for bonus points let upstream know).